dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -36,14 +35,25 @@ RUN ln -s -f /usr/bin/gcc-8 /usr/bin/arm-linux-gnueabihf-gcc
 RUN ln -s -f /usr/bin/gcc-8 /usr/bin/aarch64-linux-gnu-gcc
 {% endif %}
 
+# TODO: Remove these lines once restore_neighbors.py is converted to Python 3
+RUN apt-get install -f -y python-dev python-pip
+RUN pip2 install --upgrade 'pip<21'
+RUN apt-get purge -y python-pip
+RUN pip2 install setuptools==40.8.0
+RUN pip2 install wheel==0.35.1
 RUN pip2 install \
          scapy==2.4.2 \
-         setuptools \
-         pyroute2==0.5.3
-RUN pip2 install \
+         pyroute2==0.5.3 \
          netifaces==0.10.7 \
          monotonic==1.5
 
+# Dependencies of restore_neighbors.py
+RUN pip3 install \
+         scapy==2.4.4 \
+         pyroute2==0.5.14 \
+         netifaces==0.10.9 \
+         monotonic==1.5
```

Comment:
Do we have python3 < 3.3 supported?  Wondering if we should use time lib directly on python 3.
https://pypi.org/project/monotonic/

Having this installed is fine to support python3 across the versions. but just to check.

Reply:
As stated above, I simply added this here to make the Python 3 environment in the container match the Python 2 environment, which I though would help you in your efforts to convert restore_neighbors.py to Python 3. With Python 3, we have Python 3.7 installed. I don't see any reason for us to need to support Python 3 < 3.7.


Please feel free to modify the dependencies as part of your restore_neighbors.py Python 3 pull request.